### PR TITLE
UI update, Fix Aircraft Carrier bug

### PR DESCRIPTION
**Pull request by lishaoxia1985**
Some update:
- Use another method for airUnit
YuneRXueRYueR

- Replace "TableBackground.png" in getBackground with "whiteDot", it's much better if you use set filter method in region
- Aircraft Carrier can not attack enemy unit now

### CODE REVIEW

File: core/src/com/unciv/ui/tilegroups/TileGroupIcons.kt [hunk 1]
```diff
@@ -49,46 +42,54 @@ class TileGroupIcons(val tileGroup: TileGroup){
...
+    }
 
-    fun newUnitIcon(unit: MapUnit?, oldUnitGroup: UnitGroup?, isViewable: Boolean, yFromCenter: Float, viewingCiv: CivilizationInfo?): UnitGroup? {
-        var newImage: UnitGroup? = null
-        // The unit can change within one update - for instance, when attacking, the attacker replaces the defender!
-        oldUnitGroup?.remove()
-
-        if (unit != null && isViewable) { // Tile is visible
-            newImage = UnitGroup(unit, 25f)
-            if (UncivGame.Current.settings.continuousRendering && oldUnitGroup?.blackSpinningCircle != null) {
```
Comment (lishaoxia1985) [inline core/src/com/unciv/ui/tilegroups/TileGroupIcons.kt:60]: I don't rewrite this function because I think it is meaningless.
Of course, it's up to you.

Comment (lishaoxia1985) [inline core/src/com/unciv/ui/tilegroups/TileGroupIcons.kt:60]: I don't rewrite Line 60-63 because I think it is meaningless.
Of course, it's up to you.

**Comment by yairm210** [comment]
I really don't like the UI update for the aircraft. I think it looks worse now.

**Comment by lishaoxia1985** [comment]
@yairm210 OK, but I think it's better than the former one :
1. In the former one, air unit icon will cover the military unit icon when the military unit exists.
2. Adding the air unit icon on the city button, and then you touch the button, the city button will move down and cover the nearest bottom tile, air unit in this city cannot attack the unit in the nearest bottom tile.
3. City button should not be relative with air unit but only show city information. by the way, if necessary the city button can show Max Aircraft Capacity.
4. Using new method, changing the air unit background or other features is very easy. For example, When all air units have moved, it is very easy to change their color. 
Beside, the code is very easy to read.

**Comment by yairm210** [comment]
I'm closing this, since after thinking over it for quite a while I don't agree with the UI change and that's the major change in here

**Pull request closed without merging**